RELEASE/scripts/autoscend/auto_equipment.ash

```diff
@@ -498,6 +410,32 @@ boolean possessEquipment(item equipment)
 	return equipmentAmount(equipment) > 0;
 }
 
+boolean possessOutfit(string outfitToCheck, boolean checkCanEquip) {
+	// have_outfit will report false if you're wearing some of the items
+	// it will only report true if you have all in inventory or are wearing the whole thing
+	// hence this now exists.
```

Comment:
Is it worth asking Mafia to fix this?

Reply:
Possibly, mafia dev's are being unusually helpful recently.